docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/git-merge-pr.md

```diff
@@ -0,0 +1,73 @@
+---
+sidebar_label: git-merge-pr
+description: Merges a specified open pull request.
```

Comment:
"Specified open" just doesn't sound right. We can ditch one word or the other and I don't think the meaning will be any less clear.

```suggestion
description: Merges an open pull request.
```